tests/example_policies/withIpBlock2/withIpBlock2-scheme.yaml

```diff
@@ -32,16 +37,25 @@ networkConfigList:
       - https://github.com/IBM/network-config-analyzer/blob/master/tests/example_policies/withIpBlock2/withIpBlock2-networkpolicy5.yaml
     expectedWarnings: 0
 
+  - name: np5-ipv6
+    networkPolicyList:
+      - withIpBlock2-networkpolicy5-IPv6.yaml
+    expectedWarnings: 0
+
 queries:
   - name: no_emptiness
     emptiness:
       - np1
       - np2
+      - np2-ipv6
+      - np5-ipv6
 
```

Comment:
Please add an `expected` value for this query

Reply:
Done